portability-transfer/src/main/java/org/dataportabilityproject/transfer/WorkerModule.java

```diff
@@ -55,6 +60,22 @@
     this.asymmetricKeyGenerator = asymmetricKeyGenerator;
   }
 
+  private static TransferExtension findTransferExtension(
```

Comment:
how come this was moved? not a big deal but I thought private methods go at the bottom of the class